### DISCUSSION TITLE
Streamlink and PM2 or something

### DISCUSSION THREAD

**Discussion by emresaracoglu** [streamlink/streamlink#4470] [General]
Hello,

I'm running on Debian's server and does the Streamlink command integrate with PM2?

Recording stops when the terminal is closed. How is it possible to avoid this?

**Comment by bastimeyer** [comment]
https://github.com/streamlink/streamlink/discussions/3923#discussioncomment-1202143

**Reply by emresaracoglu** [reply]
Great. Thank you again @bastimeyer
